### DISCUSSION TITLE
Deploy - transact (not payable) error when trying to deploy  to Sepolia on Metamask for Simple Storage Section Lesson 12

### DISCUSSION THREAD

**Discussion by braveheartwilliam** [Cyfrin/foundry-full-course-cu#3135] [General]
Have the "Environment" in Remix set to Injected Provider -= Metamask and shows Sepolia chain id 11155111 network.  But Deploy button inactive and when cursor over the button get message "Deploy - transact (not payable)".  I added Sepolia Testnet - Metamask as suggested in another post but I can't select it in the dropdowns.  I'm using

**Comment by braveheartwilliam** [comment]
I was working in Brave.  The Deploy button was inactive.  Opened Remix in Edge and uploaded same .sol file.  Compiled and chose Integrated and the Deploy button was active.  So the Brave browser was apparently the problem.

**Reply by EngrPips** [reply]
That is interesting. I am glad you figured it out.
